src/features/simd.md

```diff
@@ -0,0 +1,151 @@
+---
+title: ' WebAssembly SIMD: Enable Fast, Parallel applications'
```

Comment:
Nit: we mostly seem to use normal sentence case for titles in features section. Also there is an extra leading space in title.

How about this?

```suggestion
title: 'Fast, parallel applications with WebAssembly SIMD'
```